x-pack/plugins/session_view/public/components/ProcessTreeNode/index.tsx

```diff
@@ -0,0 +1,313 @@
+            id="xpack.sessionView.childProcesses"
+            defaultMessage="Child processes"
+          />
+          <EuiIcon css={styles.buttonArrow} size="s" type={getExpandedIcon(childrenExpanded)} />
+        </EuiButton>
+      );
+    }
+
+    if (alerts.length) {
+      buttons.push(
+        <EuiButton
+          key="alert-button"
+          css={styles.getButtonStyle(ButtonType.alerts)}
+          onClick={() => setAlertsExpanded(!alertsExpanded)}
+          data-test-subj="processTreeNodeAlertButton"
+        >
+          <FormattedMessage id="xpack.sessionView.alerts" defaultMessage="Alerts" />
+          <EuiIcon css={styles.buttonArrow} size="s" type={getExpandedIcon(alertsExpanded)} />
+        </EuiButton>
+      );
+    }
+
+    return buttons;
+  };
+
+  const renderSessionLeader = () => {
+    const { user, process: processDetail } = process.getDetails();
+    const { name, args } = processDetail;
+    const sessionIcon = !!tty ? 'consoleApp' : 'compute';
+
+    return (
+      <>
+        <EuiIcon type={sessionIcon} /> <b css={styles.darkText}>{name || args[0]}</b>
+        &nbsp;
+        <FormattedMessage id="xpack.sessionView.startedBy" defaultMessage="started by" />
+        &nbsp;
+        <EuiIcon type="user" />
+        &nbsp;
+        <b css={styles.darkText}>{user.name}</b>
+      </>
+    );
+  };
+
+  // TODO: not customizable for now (cmd previously offered a template string to render)
+  const template = () => {
+    const {
+      args,
+      executable,
+      working_directory: workingDirectory,
+      exit_code: exitCode,
+    } = process.getDetails().process;
+    if (hasExec) {
+      return (
+        <span ref={textRef}>
+          <span css={styles.workingDir}>{workingDirectory}</span>&nbsp;
+          <span css={styles.darkText}>{args[0]}</span>&nbsp;
+          {args.slice(1).join(' ')}
+          {exitCode && <small> [exit_code: {exitCode}]</small>}
+        </span>
+      );
+    } else {
+      return (
+        <span ref={textRef}>
+          <span css={styles.workingDir}>{workingDirectory}</span>&nbsp;
+          <span css={styles.darkText}>{executable}</span>&nbsp;
+        </span>
+      );
+    }
+  };
+
+  const renderProcess = () => {
+    return (
+      <span>
+        {process.isUserEntered() && (
+          <EuiIcon
+            data-test-subj="processTreeNodeUserIcon"
+            css={styles.userEnteredIcon}
+            type="user"
+          />
+        )}
+        {hasExec ? (
+          <EuiIcon data-test-subj="processTreeNodeExecIcon" type="console" />
+        ) : (
+          <EuiIcon type="branch" />
+        )}
+        {template()}
+      </span>
+    );
+  };
+
+  const renderRootEscalation = () => {
+    const { user } = processDetails;
+    const { parent } = processDetails.process;
+
+    if (user.name === 'root' && user.id !== parent.user.id) {
+      return (
+        <EuiButton
+          data-test-subj="processTreeNodeRootEscalationFlag"
```

Comment:
`xpack.sessionViewer.processTreeNodeRootEscalationFlag`. This may seem like overkill, sorry about that 😓 , just trying to keep things consistent across the app

Reply:
Thanks, @michaelolo24 , I don't think it's overkill, it makes sense, we will review all the `data-test-subj` flags in the session view plugin 

cc @animehart